api/messages/publish.proto

```diff
@@ -15,15 +15,17 @@ message PublishRequest {
  // matches the current uuid of the shipper process. The uuid identifies
  // the current shipper process, and is updated when the shipper restarts.
  // Its current value is returned in every shipper API reply.
+ //
  // A uuid in a PublishRequest is used for enforcing at-least-once delivery
  // guarantees: inputs may include the known shipper uuid with their request,
  // ensuring it will be rejected if the shipper restarts. In this case the
  // input should rewind to the last known-good position in its data sequence.
+ //
  // Note that this issue only arises during error states, since Agent only
  // restarts the shipper when its process is terminated or nonresponsive.
- string uuid = 2;
+ string uuid = 1;
 
- repeated Event events = 1;
+ repeated Event events = 2;
```

Comment:
Nobody is using this yet so we can clean this up, but this is a breaking change that functionally doesn't have any effect besides being nicer to read :)

Reply:
The integer changes don't affect the wire format, so technically not breaking from protobufs perspective per https://developers.google.com/protocol-buffers/docs/proto3#updating:

> int32, uint32, int64, uint64, and bool are all compatible – this means you can change a field from one of these types to another without breaking forwards- or backwards-compatibility. If a number is parsed from the wire which doesn't fit in the corresponding type, you will get the same effect as if you had cast the number to that type in C++ (for example, if a 64-bit number is read as an int32, it will be truncated to 32 bits)